README.md

```diff
@@ -163,11 +164,6 @@ Started 1 cron jobs
 (...)
 ```
 
-
```

Comment:
Previous section "Test du cron" (l.157) may also be deleted. Personaly I never used it, and import data is now automatically tested.